src/main/java/me/aborozdykh/amazonreview/service/impl/DataReaderImpl.java

```diff
@@ -0,0 +1,62 @@
+package me.aborozdykh.amazonreview.service.impl;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TimeZone;
+import me.aborozdykh.amazonreview.entity.dto.ReviewRequestDto;
+import me.aborozdykh.amazonreview.service.DataReader;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Andrii Borozdykh
+ */
+@Service
+public class DataReaderImpl implements DataReader {
+    @Value("${reviewsFile.path}")
```

Comment:
what if I want to get reviews from several files?

Reply:
Ok, I will change it.